extra_tests/snippets/stdlib_ctypes.py

```
@@ -448,4 +448,14 @@ def get_win_folder_via_ctypes(csidl_name: str) -> str:
 array3[0:3] = [7, 8, 9]
 assert list(array3) == [7, 8, 9]
 
+
+# An array type carries the size of its buffer, so one too large to allocate
+# must raise instead of aborting.
+try:
+    (ctypes.c_char * (2**60))()
+except MemoryError:
+    pass
+else:
+    assert False, "an unallocatable array was created"
```

Ruff (0.16.1)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)